tests/github-trending.coffee

```diff
@@ -0,0 +1,57 @@
+Helper = require('hubot-test-helper')
+helper = new Helper('./../scripts/github-trending.js')
+
+Promise = require('bluebird')
+co = require('co')
+expect = require('chai').expect
+
+describe 'github-trending', ->
+  beforeEach ->
+    @room = helper.createRoom(httpd: false)
+
+  context "user asks for the trending repositories", ->
+    beforeEach ->
+      co =>
+        yield @room.user.say 'john', "hubot trending"
+        yield new Promise.delay(500)
+
+    it 'gets the trending projects', ->
+      expect(@room.messages).to.eql [
+        ['john', "hubot trending"]
+        ['hubot', "/dthree/cash - /FreeCodeCamp/FreeCodeCamp - /facebook/draft-js - /santinic/how2 - /copy/v86 - /ryanflorence/react-project - /ImagicalMine/ImagicalMine - /VPenkov/okayNav - /IBM-Swift/Kitura - /allenwong/30DaysofSwift - /dustturtle/RealReachability - /dthree/vorpal - /ampproject/amphtml - /legomushroom/mojs - /callmecavs/bricks.js - /xiepeijie/SwipeCardView - /gabrielrcouto/php-terminal-gameboy-emulator - /codrops/Animocons - /substance/substance - /Freelander/Android_Data - /Ramotion/circle-menu - /Zepo/MLeaksFinder - /veniversum/git-visualizer - /AllThingsSmitty/css-protips - /RFStorm/mousejack"]
```

Comment:
@SpoonBoy Could we change this into `dthree/cash, FreeCodeCamp/FreeCodeCamp ...`? It's a detail but it'd be a bit less verbose. What do you think?